ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -129,10 +177,15 @@ message PartyDetails {
 
   // Human readable name associated with the party. Caution, it might not be
   // unique.
+  // Use of this field is discouraged. Consider using modifiable participant local party metadata instead.
   // Optional
   string display_name = 2;
```

Comment:
```suggestion
  // Human readable name associated with the party at allocation time. 
  // Caution, it might not be unique.
  // Use of this field is discouraged. Consider using the `local_metadata` field instead.
  // Optional, Read-Only
  string display_name = 2;
```